scipy/stats/_continuous_distns.py

```diff
@@ -2306,6 +2306,47 @@ def _entropy(self, c):
 gompertz = gompertz_gen(a=0.0, name='gompertz')
 
 
+class gamma_gompertz_gen(rv_continuous):
+    """ A Gamma/Gompertz continuousrandom variable
+    
+    %(before_notes)s
+    
+    Notes
+    -----
+    The probability density function for `gamma_gompertz` is::
+    
+        gamma_gompertz.pdf(x, c, beta) = c * exp(x) * (beta ** c) / ((beta - 1 + exp(x)) ** (c + 1))
+    
+    for ``x >= 0``, ``c > 0``, ``beta > 0``.
+    
+    `gamma_gompertz` takes ``c`` and ``beta`` as shape parameters
+    
+    %(after_notes)s
+    
+    References
+    ----------
+    https://en.wikipedia.org/wiki/Gamma/Gompertz_distribution    
+    
+    %(example)s
+    """
+        
+    def _logsf(self, x, c, beta):
+        return c * (np.log(beta) - np.where(np.isfinite(sc.expm1(x)), np.log(beta + sc.expm1(x)), x))
```

Comment:
This ``np.where`` usage needs a comment. Haven't looked in detail, but it looks a bit odd.